test/testutil/fake_random.c

```diff
@@ -12,15 +12,18 @@
 #include <openssl/core_names.h>
 #include <openssl/rand.h>
 #include <openssl/provider.h>
+#include "../include/crypto/evp.h"
+#include "../../crypto/evp/evp_local.h"
 #include "../testutil.h"
 
 typedef struct {
-    int (*cb)(unsigned char *out, size_t outlen);
+    int (*cb)(unsigned char *out, size_t outlen,
```

Comment:
Should this be a typedef in testutil.h ?

Reply:
I've added it.